client/view/showStats.js

```diff
@@ -0,0 +1,141 @@
+    if (result === 0) {
+      return '0';
+    } else if (result > 5) {
+      return result + '%';
+    }
+    return '1px';
+  }
+
+  const coverageCell = document.createElement('td');
+  coverageCell.classList.add('BrowsersStat__td');
+  coverageCell.innerHTML = coveragePercentageHtmlString(coverage);
+  coverageCell.classList.add('BrowsersStat__td--coverage');
+  coverageCell.setAttribute('style', `--c:${coveragePercentageCssString(coverage)}`);
+  return coverageCell;
+}
+
+function _createVersionCell(version) {
+  const versionCell = document.createElement('td');
+  versionCell.classList.add('BrowsersStat__td');
+  versionCell.innerHTML = version;
+  return versionCell;
+}
+
+export function updateBrowsersStats(data) {
+  const element = document.getElementById('browsers-stats');
+
+  const table = document.createElement('table');
+  table.classList.add('BrowsersStat__table');
+
+  data.forEach(({id, name, versions: versionsInput}) => {
+
+    const versions = Object.entries(versionsInput)
+      .sort(([, coverageA], [, coverageB]) => coverageB - coverageA)
+      .map(([ version, coverage ]) => {
+        return {
+          version,
+          coverage,
+        }
+      });
+
+    const tBody = document.createElement('tbody');
+    const tr = document.createElement('tr');
+    tr.classList.add('.BrowsersStat_tr');
+
+    const iconCell = document.createElement('td');
+    iconCell.classList.add('BrowsersStat__td');
+    const iconElem = document.createElement('img');
+    iconElem.classList.add('BrowsersStat__icon');
+    iconElem.src = `/assets/browser_logos/${id}.svg`;
+    iconCell.setAttribute('rowspan', versions.length);
+    iconCell.appendChild(iconElem);
+    tr.appendChild(iconCell);
+
+    const nameCell = document.createElement('td');
+    nameCell.classList.add('BrowsersStat__td');
+    const nameLink = document.createElement('a');
+    nameLink.classList.add('BrowsersStat__link', 'Link');
+    nameLink.href = _getWikipediaLink(id);
+    nameLink.rel = "noreferrer noopener";
+    nameLink.target = "_blank";
+    nameCell.setAttribute('rowspan', versions.length);
+    nameLink.innerHTML = name;
+    nameCell.appendChild(nameLink);
+    tr.appendChild(nameCell);
+
+    tr.appendChild(_createVersionCell(versions[0].version));
+
+    tr.appendChild(_createCoverageCell(versions[0].coverage));
+
+    tBody.appendChild(tr);
+
+    versions.slice(1).forEach((item) => {
+      const {version, coverage} = item;
+      const versionTr = document.createElement('tr');
+
+      versionTr.appendChild(_createVersionCell(version));
+
+      versionTr.appendChild(_createCoverageCell(coverage));
+
+      tBody.appendChild(versionTr);
+    })
+    table.appendChild(tBody);
+  });
+
+  element.innerHTML = '';
+  element.appendChild(table);
+}
+
+export function updateToolsVersions({ browserslist, caniuse }) {
+  const canIUseElement = document.getElementById('can_i_use_version');
```

Comment:
Let’s always use classes for code unification

Reply:
But it isn't semantic... Class is for appearance of component (colors and outher), is for connect html with js.

Reply:
index.html:73
<img width="762" alt="image" src="https://user-images.githubusercontent.com/27442574/182659168-9cefd98c-4540-49b8-ade9-0cc4b49d546a.png">